decoder/vaapidecoder_h264_dpb.cpp

```diff
@@ -55,7 +55,7 @@ static uint32_t roundLog2(uint32_t value)
 {
     uint32_t ret = 0;
     uint32_t valueSquare = value * value;
-    while ((1 << (ret + 1)) <= valueSquare)
+    while ((uint32_t)(1 << (ret + 1)) <= valueSquare)
```

Comment:
since compiler think 1 << (ret + 1) is singed int, this maybe overflow. 
could we use this:

uint32_t one = 1;
while ((one << (ret + 1)) <= valueSquare)